src/index.css

```diff
@@ -1,13 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Vollkorn:ital,wght@0,400;0,500;0,600;0,669;0,700;0,800;0,900;1,400;1,500;1,600;1,669;1,700;1,800;1,900&display=swap');
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
+        'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
+        'Helvetica Neue', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+        monospace;
+}
+
+h1 {
+    font-family: 'Abril Fatface';
+    text-align: center;
+    font-size: 60px;
+}
+
+.explanations {
+    font-style: italic;
+    font-family: 'Vollkorn';
+    font-size: 16px;
+}
+
+.next {
+    padding: 6px 0 6px 0;
+    font: bold 20px Arial;
+    background: #478bf9;
+    color: #fff;
+    border-radius: 2px;
+    width: 100px;
+    border: none;
+}
+
+.wrongAnswer {
+    color: #de5a5a;
+}
```

Comment:
tu devrais pas mettre ton css dans ce fichier. tu peux créer à côté de ton composant un fichier `style.module.css`dans lequel le mettre.
Puis tu `import s from ./style.module.css` dans ton composant. 
Et tu ajoutes les classes dans ton composant de la manière suivante `className={s.class}`

Reply:
j'ai pas compris, y a déjà un fichier css dédié aux components. Dans celui là, j'ai mis les choses que je compte réutiliser dans toutes les étapes pour avoir une certaine homogénéité et pas refaire 50 fois le css

Reply:
l'homogénéité elle vient de la réutilisation de tes composants plutôt non ? :p 
Si tu veux avoir par exemple le même bouton partout sur ton site, il est pertinent de faire un composant Button qui aura son propre CSS et que tu réutiliseras partout